sdk/program/Cargo.toml

```diff
@@ -85,3 +82,6 @@ crate-type = ["cdylib", "rlib"]
 
 [features]
 default = []
+
+[package.metadata.cargo-udeps.ignore]
+development = ["anyhow"]
```

Comment:
Would it make sense to leave a comment explaining the reasoning for the exception?

Reply:
will do. :)